src/shogun/features/CombinedFeatures.cpp

```diff
@@ -92,9 +77,9 @@ bool CCombinedFeatures::check_feature_obj_compatibility(CCombinedFeatures* comb_
 				SG_UNREF(f1);
 				SG_UNREF(f2);
 				SG_INFO("not compatible, combfeat\n")
-				comb_feat->list_feature_objs();
+				SG_INFO(comb_feat->to_string().c_str())
```

Comment:
I am not sure `SG_INFO` acutally works this way. Wouldn't it have to be
`SG_INFO("%s\n", comb_feat->to_string.c_ctr());`?

Reply:
Did you test this and look at the output?

Reply:
I run just UT, but it doesn't cover that part of code. anyway I run: in 
TEST(CombinedFeaturesTest,test_array_operations) thi spart of code
std::cout<<"\n"<<comb_feat->to_string().c_str()<<"\n";

this is output:

CombinedFeatures(cache_size=0 ,feature_array=DynamicObjectArray(...) ,num_subscriptions={function},num_vec=3 ,preproc=DynamicObjectArray(...) ,properties=0 ,subset_stack=SubsetStack(...) )

Of course I can remove list_feature_obj() methode from the code at all but as I understand I should replace it with to_string().

Reply:
Hi ktiefe. Please check that the output ends with a newline character. I suspect that if you did `SG_INFO(comb_feat->to_string().c_str()); SG_INFO("Hello, World\n");` the output is going to be concatenated as follows
```
CombinedFeatures(cache_size=0 ,feature_array=DynamicObjectArray(...) ,num_subscriptions={function},num_vec=3 ,preproc=DynamicObjectArray(...) ,properties=0 ,subset_stack=SubsetStack(...) )Hello, World
```
If that's the case, what @karlnapf suggested seems to be the proper approach.